src/scripts/lib/server/utils/server-config/setup-prompts.js

```diff
@@ -4,7 +4,14 @@ module.exports = [{
   type: 'input',
   name: 'clientData',
   message: 'Client data directory',
-  default: 'C:/Program Files (x86)/World of Warcraft/Data',
+  default: function() {
+    if (process.platform === 'win32') {
+      return 'C:/Program Files (x86)/World of Warcraft/Data'
+    }
+    else {
```

Comment:
Could you add the brace and the else on the same line? Other than that, looks stellar! :+1: